README.md

```diff
@@ -34,12 +34,7 @@ $ conda install -c conda-forge tiledb
 
 (see links below for Python, R, and other API installation instructions)
 
-Alternatively, you can use the Docker image we provide:
-
-```bash
-$ docker pull tiledb/tiledb
-$ docker run -it tiledb/tiledb
-```
+Alternatively, you can use the [Dockerfile we provide.](examples/Dockerfile/Dockerfile)
```

Comment:
I would remove this line entirely. I'm not sure the `Dockerfile` is something we should advertise to external customers.

Reply:
I'm ok with this, we've had various requests for baseline "how to build container" and "what are ubuntu requirements" that this can help with